### ISSUE TITLE
Command Suggestion: /crew

### ISSUE THREAD

**Issue by DUFCLiam** [TotalFreedom/TotalFreedomMod#1642]
/crew <create | leave | join | ranks <create |  remove | edit>  | admin <kick | accept | deny | promote | demote> | options <tag>>

**Comment by DUFCLiam** [comment]
samennis1 created the idea!

**Comment by DUFCLiam** [comment]
samennis1 did not help!

**Comment by samennis1** [comment]
My idea <3

**Comment by RobinGall2910** [comment]
Nah, doesn't seem useful anyways.... Sounds like some plugin and, if you are going to add it, add a toggle for that command to be enabled or disabled in a config, or a command... or whereever

**Comment by DUFCLiam** [comment]
@RobinGall2910, It is not a plugin.

**Comment by RobinGall2910** [comment]
@DUFCLiam Read better next time, it sounds like some plugin that I heard of, not an actual plugin.
